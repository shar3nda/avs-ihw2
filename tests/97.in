mO!ck)gE7v>Z0<Ci"#11GSznK';|z`87W"M,xpbK{g.-XU+ER{QK*1\Y_)@H,Im^ag	$rjHXb&t.[=