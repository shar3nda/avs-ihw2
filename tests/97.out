Lowercase: 946, uppercase: 965